src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqBuildReferenceTaxonomy.java

```diff
@@ -0,0 +1,111 @@
+
+        //Parse RefSeq catalog to determine taxonomic ID's of accession keys in accessionToNameAndLength
+        Set<String> accessionsNotFound = null;
+        if (refseqCatalogPath != null) {
+            final BufferedReader refseqCatalogStreamReader = PSBuildReferenceTaxonomyUtils.getBufferedReaderGz(refseqCatalogPath);
+            accessionsNotFound = PSBuildReferenceTaxonomyUtils.parseCatalog(refseqCatalogStreamReader, accessionToNameAndLength, taxIdToProperties, false, null);
+            PSBuildReferenceTaxonomyUtils.closeReader(refseqCatalogStreamReader);
+        }
+
+        //Parse Genbank catalog
+        if (genbankCatalogPath != null) {
+            final BufferedReader genbankCatalogStreamReader = PSBuildReferenceTaxonomyUtils.getBufferedReaderGz(genbankCatalogPath);
+            accessionsNotFound = PSBuildReferenceTaxonomyUtils.parseCatalog(genbankCatalogStreamReader, accessionToNameAndLength, taxIdToProperties, true, accessionsNotFound);
+            PSBuildReferenceTaxonomyUtils.closeReader(genbankCatalogStreamReader);
+        }
+        if (accessionsNotFound != null && !accessionsNotFound.isEmpty()) {
+            PSUtils.logItemizedWarning(logger, accessionsNotFound, "Did not find entries in the catalog for the following reference accessions");
+        }
+
+        //Get the scientific name of every taxonomic node (even the ones not in the reference)
+        final BufferedReader namesStreamReader = PSBuildReferenceTaxonomyUtils.getBufferedReaderTarGz(taxdumpPath, "names.dmp");
+        PSBuildReferenceTaxonomyUtils.parseNamesFile(namesStreamReader, taxIdToProperties);
+        PSBuildReferenceTaxonomyUtils.closeReader(namesStreamReader);
+
+        //Gets the taxonomic rank (e.g. family, order, genus, species, etc.) and parent of each node
+        final BufferedReader nodesStreamReader = PSBuildReferenceTaxonomyUtils.getBufferedReaderTarGz(taxdumpPath, "nodes.dmp");
+        final Collection<String> taxNotFound = PSBuildReferenceTaxonomyUtils.parseNodesFile(nodesStreamReader, taxIdToProperties);
+        PSUtils.logItemizedWarning(logger, taxNotFound, "Did not find entry from reference sequence names or the names file for following some tax ID's. Setting name to tax_<tax ID>");
+        PSBuildReferenceTaxonomyUtils.closeReader(nodesStreamReader);
+
+        //Build the taxonomic tree and a map from accession ID's to taxonomic ID's
+        logger.info("Building taxonomic database...");
+        final PSTree tree = PSBuildReferenceTaxonomyUtils.buildTaxonomicTree(taxIdToProperties);
+        final Map<String, String> accessionToTaxId = PSBuildReferenceTaxonomyUtils.buildAccessionToTaxIdMap(taxIdToProperties);
+
+        //Write output
+        PSBuildReferenceTaxonomyUtils.writeTaxonomyDatabase(outputPath, new PSTaxonomyDatabase(tree, accessionToTaxId));
+    }
+
+    @Override
+    public void traverse() {}
```

Comment:
It seems a little silly to have this be a `GATKTool` with an empty `traverse` method. Would it be hard to make it into a `CommandLineProgram` instead? The only thing I can see that you're using from `GATKTool` is the reference initialization, which I think is only a line of code.